src/ImageSharp/Common/Extensions/StreamExtensions.cs

```diff
@@ -82,5 +83,22 @@ public static void Write(this Stream stream, IManagedByteBuffer buffer)
         {
             stream.Write(buffer.Array, 0, buffer.Length());
         }
+
+#if NET472 || NETSTANDARD1_3 || NETSTANDARD2_0
+        // This is a port of the CoreFX implementation: https://github.com/dotnet/coreclr/blob/c4dca1072d15bdda64c754ad1ea474b1580fa554/src/System.Private.CoreLib/shared/System/IO/Stream.cs#L770
```

Comment:
I didn't see a 3rd party license file in the repo root. Do you normally just list that inline (given this is a copy from CoreFX, it is technically MIT).

Reply:
Inline is fine for now but we should certainly add a file for future work.

Reply:
Updated the comment to explicitly call out the MIT license and logged https://github.com/SixLabors/ImageSharp/issues/856 to track adding a 3rd party notices file.